### PR TITLE
Add a new sample: key-vault-secret-client.

### PR DESCRIPTION
**Pull request by chenrujun**
1. This sample is copied from "single-property-source".
2. Change package to "com.azure.spring.keyvault.secrets.sample.client".
3. In SampleApplication, Use SecretClient to get the secret value.
4. Delete unnecessary dependencies: spring-boot-starter-actuator and spring-boot-starter-web.
5. Update root pom and root README.md
6. Update the value stored in Azure Key Vault Secret, to make the output more clear.

### CODE REVIEW

File: keyvault/spring-cloud-azure-starter-keyvault-secrets/key-vault-secret-client/pom.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <artifactId>spring-cloud-azure-starter-keyvault-secrets</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-actuator</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
+    </dependency>
```
Comment (backwind1233) [inline keyvault/spring-cloud-azure-starter-keyvault-secrets/key-vault-secret-client/pom.xml:32]: can we delete these two dependencies here?

File: keyvault/spring-cloud-azure-starter-keyvault-secrets/key-vault-secret-client/README.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
+# Using Key Vault Secrets With Spring Cloud Azure Key Vault Secrets Starter
+
+This sample illustrates the simplest usage of `spring-cloud-azure-starter-keyvault-secrets`. 
+To learn all features, please refer to [reference doc](https://microsoft.github.io/spring-cloud-azure/current/reference/html/index.html#secret-management).
```
Comment (backwind1233) [inline keyvault/spring-cloud-azure-starter-keyvault-secrets/key-vault-secret-client/README.md:4]: so, you think the link is ok with "current"?

File: pom.xml [hunk 3]
```diff
@@ -58,6 +58,7 @@
     <module>keyvault/azure-securtiy-keyvault-jca/run-with-command-line-client-side</module>
     <module>keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-client-side</module>
     <module>keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side</module>
+    <module>keyvault/spring-cloud-azure-starter-keyvault-secrets/key-vault-secret-client</module>
```
Comment (backwind1233) [inline pom.xml:61]: I think you should also update the root readme and root pom

File: keyvault/spring-cloud-azure-starter-keyvault-secrets/key-vault-secret-client/pom.xml (hunk 1 above)
Comment (chenrujun) [reply keyvault/spring-cloud-azure-starter-keyvault-secrets/key-vault-secret-client/pom.xml]: Updated.

File: keyvault/spring-cloud-azure-starter-keyvault-secrets/key-vault-secret-client/README.md (hunk 2 above)
Comment (chenrujun) [reply keyvault/spring-cloud-azure-starter-keyvault-secrets/key-vault-secret-client/README.md]: No, just keep align with other samples in this PR.
If we decide to change it, we need to change it in the whole repo.

File: pom.xml (hunk 3 above)
Comment (chenrujun) [reply pom.xml]: Updated.

**Review by backwind1233 — APPROVED** [review]
LGTM

**Pull request merged**